extensions/wikia/AdEngine/js/lookup/adapter/appnexusPlacements.js

```diff
@@ -0,0 +1,32 @@
+define('ext.wikia.adEngine.lookup.adapter.appnexusPlacements', [
+	'ext.wikia.adEngine.utils.adLogicZoneParams',
+	'wikia.window'
+], function (zoneParams, win) {
+	'use strict';
+
+	var placements = {
+		mercury: {
+			entertainment: isDevEnviroment() ? '9412980' : '9412992',
```

Comment:
I have no idea what happen here and what represents this 'magic numbers', maybe extract to constans, to simplify it? 
ENTERTAINMENT_DEV_ID etc.